src/runner/driver-test-runner.js

```diff
@@ -66,11 +66,30 @@ export class DriverTestRunner {
    */
   async pressKeysToToggleSetting(sequence, desiredResponse) {
     let unknownCollected = '';
-    // there are 2 modes, so we will try pressing mode switch up to twice
-    for (let triesRemain = 2; triesRemain > 0; triesRemain--) {
+    let unexpectedAllowance = 2;
```

Comment:
given what we witnessed, I might like `3` here instead.

if for some reason the browser became "awake" after the insert+space, in the "modeSwitch" window, and only read it's title/etc, we would then only >really< try the switch once.

Reply:
Sounds good